barberease/usuarios/models.py

```diff
@@ -17,4 +19,7 @@ def __str__(self):
         return self.nome
     
 
- 
+class SenhaRedefinicao(models.Model):
```

Comment:
Gabriel, cada classe criada no models é uma tabela criada no banco, no processo de redefinição de senha não é preciso criar mais uma tabela. para alterar uma senha voce tem que usar o models do usuario na alteração. Peço que modifique isso na seu codigo. Segue um link para ajudar [https://www.tutorialspoint.com/how-to-change-password-of-users-in-django](https://www.tutorialspoint.com/how-to-change-password-of-users-in-django). Tente usar esse metodo ou procure desde que a senha tem hash e nao crie outra tabela no bd